src/libstd/io/mod.rs

```diff
@@ -205,16 +205,15 @@
 //!
 //! Last, but certainly not least, is [`io::Result`]. This type is used
 //! as the return type of many `std::io` functions that can cause an error, and
-//! can be returned from your own functions as well. Many of the examples in this
-//! module use the [`try!`] macro:
+//! can be returned from your own functions as well.
 //!
```

Comment:
My inclination is to say that this PR should be about a fairly mechanical translation, so keeping it here is fine.